resources/views/account/partials/get-started.blade.php

```diff
@@ -48,17 +48,17 @@
                         @include('account.partials.get-started-checklist-item', [
                             'number' => '2',
                             'condition' => $user->induction_completed,
-                            'title' => 'Read through our General Induction',
-                            'link' => route('account.induction.show', $user->id),
-                            'description' => 'Our General Induction includes important information about being a member, how to get to the Hackspace, and other important information.'
+                            'title' => 'Get a General Induction',
+                            'link' => route('general-induction.show', $user->id),
+                            'description' => 'Come along to an open evening and get a General Induction which includes a tour and important information about being a member.'
                         ])
                         @include('account.partials.get-started-checklist-item', [
                             'number' => '3',
                             'condition' => $user->keyFob(),
-                            'title' => 'Visit the Hackspace to finish setting up',
+                            'title' => 'Get an access method set up',
                             'link' => 'https://www.hacman.org.uk/visit-us/',
```

Comment:
Link probably could do with updating after the change. Makes sense to point it to the account edit page, where people can manage their access?

```suggestion
                            'link' => route('account.edit', $user->id),
```